install_config/adding_hosts_to_existing_cluster.adoc

```diff
@@ -133,7 +135,9 @@ endif::[]
 
 [IMPORTANT]
 ====
-The recommended maximum number of nodes is 2000.
+See the
+xref:../../scaling_performance/cluster_limits.adoc#scaling-performance-cluster-limits[cluster
```

Comment:
@bfallonf - wrong path. Should have been :../scaling_performance/cluster_limits.adoc